public/class-lti-platform-public.php

```diff
@@ -243,7 +292,9 @@ public function message($deeplink = false)
                 $url = "{$tool->messageUrl}{$link_atts['url']}";
             } elseif (strpos($link_atts['url'], $tool->messageUrl) === 0) {
                 $url = $link_atts['url'];
-            } else {
+            }elseif (str_contains(parse_url($tool->messageUrl)['path'], '/mod/curriki')) {  
```

Comment:
what this block do ?

Reply:
Previously at the time of deep linking the launch url was supposed to contain "/content" in it but at the time of performing activity afterwards we had to remove "/content" from launch url  tsugi/mod/curriki/content. I have added an else if block to check if the request is to curriki studio then it will consider it no matter it contains "/content" or not.

Reply:
else block is executed if no condition is matched and it is only returning an error nothing else.